requre/__init__.py

```diff
@@ -0,0 +1,3 @@
+from requre.constants import VERSION
+
+_ = VERSION
```

Comment:
See [ogr/__init__.py](https://github.com/packit-service/ogr/blob/master/ogr/__init__.py#L48) how to deal with this.